bdk-ffi/src/bdk.udl

```diff
@@ -446,6 +446,9 @@ interface Address {
   [Throws=BdkError]
   constructor(string address);
 
+  [Name=from_script, Throws=BdkError]
+  constructor(Script script_pub_key, Network network);
```

Comment:
Let's name the argument `script`, as per the [rust-bitcoin method signature](https://docs.rs/bitcoin/0.29.2/src/bitcoin/util/address.rs.html#770-776)

Reply:
done.